src/features/common/Layout/PlanetPopup/index.tsx

```diff
@@ -0,0 +1,104 @@
+import CancelIcon from '../../../../../public/assets/images/icons/CancelIcon';
+import tenantConfig from '../../../../../tenant.config';
+
+const tenantConfiguration = tenantConfig();
+
+interface Props {}
+
+export default function index({}: Props): ReactElement {
+  const [hidePlanetModalNew, setHidePlanetModalNew] = React.useState(false);
+  const [openModal, setModalOpen] = React.useState(true);
+  const [userLang, setUserLang] = React.useState('');
+  const [countryCode, setCountryCode] = React.useState('');
+  React.useEffect(() => {
+    if (typeof window !== 'undefined') {
+      let userLang = localStorage.getItem('language');
+      let countryCode = localStorage.getItem('countryCode');
+
+      //remove old key
+      let hidePlanetModalOld = localStorage.getItem('hidePlanetModal');
+      if(hidePlanetModalOld) {
+        localStorage.removeItem('hidePlanetModal');
+      }
+
+      let hidePlanetModalNew = localStorage.getItem('hidePlanetModalNew');
+      if (hidePlanetModalNew === 'true') {
+        setHidePlanetModalNew(true);
+      }
+      setUserLang(userLang);
+      setCountryCode(countryCode);
+    }
+  }, []);
+
+  const closePlanetModal = () => {
+    setHidePlanetModalNew(true);
+    setModalOpen(false);
+    localStorage.setItem('hidePlanetModalNew', 'true');
+  };
+  return (
+    <>
+      {tenantConfiguration.tenantName === 'planet' &&
+        (countryCode === 'DE' || userLang === 'de') &&
+        hidePlanetModalNew !== true && (
+          <Modal
+            style={{
+              display: 'flex',
+              height: '100%',
+              width: '100%',
+              justifyContent: 'center',
+              alignItems: 'center',
+            }}
+            open={openModal}
+            onClose={closePlanetModal}
+            aria-labelledby="simple-modal-title"
+            aria-describedby="simple-modal-description"
+          >
+            <div
+              style={{
+                backgroundColor: 'white',
+                minHeight: '200px',
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                padding: '30px',
+                borderRadius: '10px',
+                maxWidth: '580px',
+                position: 'relative',
+                flexDirection: 'column',
+              }}
+            >
+              <h2 style={{ fontWeight: 'bold' }}>
+              Transparenzbericht 2020 erschienen
+              </h2>
+              <p style={{ margin: '16px auto' }}>
+               Acht Jahre mit uneingeschränkten Bestätigungsvermerken - Erleben Sie Plant-for-the-Planet in testierten Zahlen, Daten und spannenden Grafiken.
+              </p>
+              <a
+                target="_blank"
+                style={{ fontWeight: 'bold', color: '#68B030' }}
+                rel="noopener noreferrer"
+                href={
+                  'https://drive.google.com/file/d/11vdq-uVS-HPECtUGEEy5EuUaBHeGpzID/view'
```

Comment:
On https://a.plant-for-the-planet.org/ the link differs from this one: https://drive.google.com/file/d/10bC4BwV0oFBQ87AX3ZdgcPVUaljFxSUn/view and on the page https://a.plant-for-the-planet.org/annual-reports/ the links even changed again. If we still use this PR on production, we maybe should adapt the link https://drive.google.com/file/d/1LJofiDcfKfQXNuj-l50aGU9bn8gjbdFb/view (and hope the file will not be adapted again that quickly :-)).

Reply:
Latest version of https://a.plant-for-the-planet.org/ is showing original link again.